visualization/react-lib/src/components/space-chart/SpaceChart.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
```

Comment:
this and tablechart.tsx are both super lean wrapper now. It makes me think if we should just get rid of it. I always confuse myself betweem spacechart and spacedigram.

Reply:
agreed- I consolidated and got rid of timechart and spacechart